toolchain/rustup.cmake

```diff
@@ -0,0 +1,20 @@
+if(${TARGET_CPU} MATCHES "x86_64")
+    set(target "x86_64-pc-windows-gnu")
+else()
+    set(target "i686-pc-windows-gnu")
+endif()
+
+ExternalProject_Add(rustup
+    DOWNLOAD_COMMAND ""
+    UPDATE_COMMAND ""
+    PREFIX rustup-prefix
+    SOURCE_DIR rustup-prefix/src
+    CONFIGURE_COMMAND ${EXEC}
+        curl -sSf https://sh.rustup.rs |
+        sh -s -- -y --default-toolchain stable --target ${target} --no-modify-path --profile minimal
+    BUILD_COMMAND ${EXEC} rustup update
+    INSTALL_COMMAND ${EXEC} cargo install cargo-c --features=vendored-openssl
```

Comment:
I think it would be preferable to install `openssl` through the Arch package instead.
This can cause issues: https://github.com/lu-zero/cargo-c/issues/309

Reply:
Seems okay with me.

I will get error anyway if cargo install without `--features=vendored-openssl` flag due to pkg-config not found